types/office-js-preview/index.d.ts

```diff
@@ -509,6 +509,32 @@ declare namespace Office {
          */
         Text,
     }
+    /**
+     * Specifies the device capability to which an add-in is requesting access.
+     *
+     * @remarks
+     *
+     * **Applications**: Excel on the web, Outlook on the web, PowerPoint on the web, and Word on the web running in Chromium-based browsers,
+     * such as Microsoft Edge and Google Chrome.
```

Comment:
Please make the same browser-first change here as elsewhere.

Reply:
Ah, thanks for catching this. Updated.